### ISSUE TITLE
object-curly-spacing false-positive with es6 import with trailing comma

### ISSUE THREAD

**Issue by leebyron** [eslint/eslint#3324]
version: 1.0.0

```
import {
  A,
  B,
} from 'module';
```

with rule:

```
"object-curly-spacing": [2, "always"]
```

reports:

```
3:4   error  A space is required before ','  object-curly-spacing
```

But I expected no error.

**Comment by eslintbot** [comment]
Thanks for the issue! We get a lot of issues, so this message is automatically posted to each one to help you check that you've included all of the information we need to help you.

Reporting a bug? Please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
1. The source code that caused the problem
1. The configuration you're using (for the rule or your entire config file)
1. The actual ESLint output complete with line numbers

Requesting a new rule? Please be sure to include:

1. The use case for the rule - what is it trying to prevent or flag?
1. Whether the rule is trying to prevent an error or is purely stylistic
1. Why you believe this rule is generic enough to be included

Requesting a feature? Please be sure to include:

1. The problem you want to solve (don't mention the solution)
1. Your take on the correct solution to problem

Including this information in your issue helps us to triage it and get you a response as quickly as possible.

Thanks!

**Comment by xjamundx** [comment]
I'll look at this today..

**Comment by xjamundx** [comment]
Weird I'm getting:

```
> require('espree').parse("import { A, B, } from 'module';", {ecmaFeatures: {modules: true, destructuring: true}})
Error: Line 1: Unexpected token }
    at throwError (/Users/jamuferguson/dev/eslint/node_modules/espree/espree.js:2070:17)
    at throwUnexpected (/Users/jamuferguson/dev/eslint/node_modules/espree/espree.js:2133:5)
    at parseNonComputedProperty (/Users/jamuferguson/dev/eslint/node_modules/espree/espree.js:2941:9)
    at parseImportSpecifier (/Users/jamuferguson/dev/eslint/node_modules/espree/espree.js:4871:16)
    at parseNamedImports (/Users/jamuferguson/dev/eslint/node_modules/espree/espree.js:4886:29)
    at parseImportDeclaration (/Users/jamuferguson/dev/eslint/node_modules/espree/espree.js:4952:40)
    at parseSourceElement (/Users/jamuferguson/dev/eslint/node_modules/espree/espree.js:5139:24)
    at parseSourceElements (/Users/jamuferguson/dev/eslint/node_modules/espree/espree.js:5192:25)
    at parseProgram (/Users/jamuferguson/dev/eslint/node_modules/espree/espree.js:5212:12)
    at Object.parse (/Users/jamuferguson/dev/eslint/node_modules/espree/espree.js:5455:19)
```

With espree 2.2.0 and eslint 1.1.0.

Are you using `babel-eslint`? There may be a bug in espree assuming trailing commas are allowed there...

**Comment by xjamundx** [comment]
1. @leebyron I think this may be a `babel-eslint` issue in the way it's handing column numbers or something?
2. The destructuring trailing-comma case works as expected with eslint (though it wasn't being tested)
3. Espree currently does not support trailing commas in `import` and ... oh wait. it looks like I was supposed to fix that last month https://github.com/eslint/espree/issues/148

**Comment by leebyron** [comment]
Yes, I'm using the `babel-eslint` parser. cc @hzoo

**Comment by hzoo** [comment]
Ok since @xjamundx has a PR https://github.com/eslint/espree/pull/186 think this might be fixed eslint side?

**Comment by hzoo** [comment]
Ok made a PR to fix after https://github.com/eslint/espree/pull/186

**Comment by xjamundx** [comment]
Awesome!
